Prevention Act;Prevents sandmining, scamming and trapping.html

```diff
@@ -0,0 +1,26 @@
+<h2>Prevention</h2>
+<h3>Sandmining</h3>
+<p>
+    Sandmining is not allowed, if a player sandmines the military may be deployed
+    and if it’s severe the criminal may be given a fine of up to 1g per chunk
+    affected. If the fine is not paid back within 2 weeks the amount fined will go up each week by 1G per chunk
+</p>
+<h3>Scams<h3>
+<p>
+   Failing to follow through with a deal without refunding or attempting to trade for an unreasonable price with
+   the intention of making an unfair profit will result in a fine of up to 16G plus 50% of the value of the scam;
+   the scammer must refund and give compensation of 16G plus 25% of the scam's value or what a judge deems appropriate.
+<p>
+<h3>Trapping<h3>
```

Comment:
Should have closing tag
```suggestion
<h3>Trapping</h3>
```